### PR TITLE
(DOCSP-16663) Backfills docs content for integrations commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills docs content for integrations commands

_Jira ticket:_ 
https://jira.mongodb.org/browse/DOCSP-16663
https://jira.mongodb.org/browse/DOCSP-16666
https://jira.mongodb.org/browse/DOCSP-16667
https://jira.mongodb.org/browse/DOCSP-16668
https://jira.mongodb.org/browse/DOCSP-16669
https://jira.mongodb.org/browse/DOCSP-16671
https://jira.mongodb.org/browse/DOCSP-16672
https://jira.mongodb.org/browse/DOCSP-16673

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/integrations/create/datadog.go [hunk 1]
```diff
@@ -69,8 +70,15 @@ func DatadogBuilder() *cobra.Command {
...
 		Use:     datadogType,
 		Aliases: []string{"datadog"},
-		Short:   "Create or update the Datadog integration.",
-		Args:    require.NoArgs,
+		Short:   "Create or update a Datadog integration for your project.",
+		Long: `The requesting API key must have the Organization Owner or Project Owner role to configure an integration with Datadog.
+
+After you integrate with Datadog, you can send metric data about your project to your Datadog dashboard. To learn more about the metrics available to Datadog, see https://www.mongodb.com/docs/atlas/tutorial/datadog-integration/.
+		
+Datadog integration is available only on M10+ clusters.`,
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/integrations/create/datadog.go:78]: Suggestion: how about "supported only on" or "available only for":
```suggestion
Datadog integration is available only for M10+ clusters.`,
```

File: internal/cli/atlas/integrations/describe.go [hunk 2]
```diff
@@ -98,12 +99,14 @@ func (opts *DescribeOpts) template() string {
...
-		Args:      require.ExactValidArgs(1),
+		Use:   "describe <integrationType>",
+		Short: "Return the details for the specified third-party integration for your project.",
+		Args:  require.ExactValidArgs(1),
+		Example: fmt.Sprintf(`  # Return the JSON-formatted details for the Datadog integration for the project with the ID 5e2211c17a3e5a48f5497de3:
+  %s integrations describe DATADOG --projectId 5e2211c17a3e5a48f5497de3 --output json`, cli.ExampleAtlasEntryPoint()),
 		ValidArgs: []string{"PAGER_DUTY", "MICROSOFT_TEAMS", "SLACK", "DATADOG", "NEW_RELIC", "OPS_GENIE", "VICTOR_OPS", "WEBHOOK", "PROMETHEUS"},
 		Annotations: map[string]string{
-			"integrationTypeDesc": "Human-readable label that identifies the service which you want to integrate with.",
+			"integrationTypeDesc": "Human-readable label that identifies the service integration. Valid values are DATADOG, OPS_GENIE, PAGER_DUTY, VICTOR_OPS, and WEBHOOK.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/integrations/describe.go:109]: Suggestion:
```suggestion
			"integrationTypeDesc": "Human-readable label that identifies the integrated service. Valid values are DATADOG, OPS_GENIE, PAGER_DUTY, VICTOR_OPS, and WEBHOOK.",
```

File: internal/usage/usage.go [hunk 3]
```diff
@@ -269,7 +272,8 @@ dbName and collection are required only for built-in roles.`
 	SubscriptionID                            = "Unique identifier of the Azure subscription in which the VNet resides."
 	GCPProjectID                              = "Unique identifier of the GCP project in which the network peer resides."
 	Network                                   = "Unique identifier of the Network Peering connection in the Atlas project."
-	APIRegion                                 = "Indicates which API URL to use, either US or EU. The integration service will use US by default."
+	APIRegion                                 = "Code that indicates which regional URL MongoDB uses to access the third-party API. Valid values are US and EU."
+	DatadogAPIRegion                          = "Code that indicates which regional URL MongoDB uses to access the Datadog API. Valid values are US, EU, US3, and US5."
```
Comment (kanchana-mongodb) [inline internal/usage/usage.go:276]: Suggestion:
```suggestion
	APIRegion                                 = "Two-letter country dode that indicates which regional URL MongoDB uses to access the third-party API. Valid values are US and EU."
	DatadogAPIRegion                          = "Two-letter country code that indicates which regional URL MongoDB uses to access the Datadog API. Valid values are US, EU, US3, and US5."
```

**Review by kanchana-mongodb — APPROVED** [review]
Couple of suggestions that feel free to ignore or apply. No need for a re-review. LGTM.

File: internal/usage/usage.go (hunk 3 above)
Comment (kanchana-mongodb) [reply internal/usage/usage.go:276]: Oops... please ignore the above comment. Just noticed that some of them aren't two letters :-) Please ignore previous comment. Thanks!

File: internal/usage/usage.go [hunk 4]
```diff
@@ -138,7 +138,10 @@ dbName and collection are required only for built-in roles.`
 	NotificationsChannelName                  = "Slack channel name. Required for the SLACK notifications type."
 	AlertConfigAPIKey                         = "Datadog API Key, Opsgenie API Key, VictorOps API key." //nolint:gosec // This is just a message not a credential
 	APIKey                                    = "API Key."
-	RoutingKey                                = "An optional field for your Routing Key."
+	DatadogAPIKey                             = "Datadog API key that allows Atlas to access your Datadog account."
```
Comment (gssbzn) [inline internal/usage/usage.go:141]: add 
`//nolint:gosec // This is just a message not a credential` at the end of the line or remove, depending on the complain from the linter

Comment (sarahsimpers) [reply internal/usage/usage.go]: @gssbzn Fixed it and the linter no longer shows errors, thank you!

File: internal/cli/atlas/integrations/describe.go [hunk 5]
```diff
@@ -98,12 +99,14 @@ func (opts *DescribeOpts) template() string {
...
-		Args:      require.ExactValidArgs(1),
+		Use:   "describe <integrationType>",
+		Short: "Return the details for the specified third-party integration for your project.",
+		Args:  require.ExactValidArgs(1),
+		Example: fmt.Sprintf(`  # Return the JSON-formatted details for the Datadog integration for the project with the ID 5e2211c17a3e5a48f5497de3:
+  %s integrations describe DATADOG --projectId 5e2211c17a3e5a48f5497de3 --output json`, cli.ExampleAtlasEntryPoint()),
 		ValidArgs: []string{"PAGER_DUTY", "MICROSOFT_TEAMS", "SLACK", "DATADOG", "NEW_RELIC", "OPS_GENIE", "VICTOR_OPS", "WEBHOOK", "PROMETHEUS"},
 		Annotations: map[string]string{
-			"integrationTypeDesc": "Human-readable label that identifies the service which you want to integrate with.",
+			"integrationTypeDesc": "Human-readable label that identifies the integrated service. Valid values are DATADOG, OPS_GENIE, PAGER_DUTY, VICTOR_OPS, and WEBHOOK.",
```
Comment (gssbzn) [inline internal/cli/atlas/integrations/describe.go:109]: the list above has
>		ValidArgs: []string{"PAGER_DUTY", "MICROSOFT_TEAMS", "SLACK", "DATADOG", "NEW_RELIC", "OPS_GENIE", "VICTOR_OPS", "WEBHOOK", "PROMETHEUS"},

which one is correct?

Comment (sarahsimpers) [reply internal/cli/atlas/integrations/describe.go:109]: The API does accept Prometheus, I thought that wasn't supported for the CLI but upon testing it looks like it is supported for describe (but no create command etc), I will add that in. Thank you!

File: internal/cli/atlas/integrations/delete.go [hunk 6]
```diff
@@ -50,13 +51,16 @@ func DeleteBuilder() *cobra.Command {
...
+		Aliases: []string{"rm"},
+		Short:   "Remove the specified third-party integration from your project.",
+		Long:    "Deleting an integration from a project removes that integration configuration only for that project. This does not affect any other project or organization's configured integrations.",
+		Args:    require.ExactValidArgs(1),
+		Example: fmt.Sprintf(`  # Remove the Datadog integration for the project with the ID 5e2211c17a3e5a48f5497de3:
+  %s integrations delete DATADOG --projectId 5e2211c17a3e5a48f5497de3`, cli.ExampleAtlasEntryPoint()),
 		ValidArgs: []string{"PAGER_DUTY", "MICROSOFT_TEAMS", "SLACK", "DATADOG", "NEW_RELIC", "OPS_GENIE", "VICTOR_OPS", "WEBHOOK", "PROMETHEUS"},
 		Annotations: map[string]string{
-			"integrationTypeDesc": "Human-readable label that identifies the service which you want to integrate with.",
+			"integrationTypeDesc": "Human-readable label that identifies the service integration to delete. Valid values are DATADOG, OPS_GENIE, PAGER_DUTY, VICTOR_OPS, and WEBHOOK.",
```
Comment (gssbzn) [inline internal/cli/atlas/integrations/delete.go:63]:
```suggestion
			"integrationTypeDesc": "Human-readable label that identifies the service integration to delete. Valid values are PAGER_DUTY, MICROSOFT_TEAMS, SLACK, DATADOG, NEW_RELIC, OPS_GENIE, VICTOR_OPS, WEBHOOK, PROMETHEUS.",
```

File: internal/cli/atlas/integrations/describe.go [hunk 7]
```diff
@@ -98,12 +99,14 @@ func (opts *DescribeOpts) template() string {
...
-		Args:      require.ExactValidArgs(1),
+		Use:   "describe <integrationType>",
+		Short: "Return the details for the specified third-party integration for your project.",
+		Args:  require.ExactValidArgs(1),
+		Example: fmt.Sprintf(`  # Return the JSON-formatted details for the Datadog integration for the project with the ID 5e2211c17a3e5a48f5497de3:
+  %s integrations describe DATADOG --projectId 5e2211c17a3e5a48f5497de3 --output json`, cli.ExampleAtlasEntryPoint()),
 		ValidArgs: []string{"PAGER_DUTY", "MICROSOFT_TEAMS", "SLACK", "DATADOG", "NEW_RELIC", "OPS_GENIE", "VICTOR_OPS", "WEBHOOK", "PROMETHEUS"},
 		Annotations: map[string]string{
-			"integrationTypeDesc": "Human-readable label that identifies the service which you want to integrate with.",
+			"integrationTypeDesc": "Human-readable label that identifies the integrated service. Valid values are DATADOG, OPS_GENIE, PAGER_DUTY, PROMETHEUS, VICTOR_OPS, and WEBHOOK.",
```
Comment (gssbzn) [inline internal/cli/atlas/integrations/describe.go:109]:
```suggestion
			"integrationTypeDesc": "Human-readable label that identifies the integrated service. Valid values are PAGER_DUTY, MICROSOFT_TEAMS, SLACK, DATADOG, NEW_RELIC, OPS_GENIE, VICTOR_OPS, WEBHOOK, PROMETHEUS.",
```

**Review by gssbzn — DISMISSED** [review]
LGTM

**Review by gssbzn — APPROVED** [review]

**Pull request merged**